### PR TITLE
Create harbor.md

### PR DESCRIPTION
**Pull request by kevin7s-io**
Added instructions for lldap config for harbor registry helm chart.

### CODE REVIEW

File: example_configs/harbor.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Using the [harbor-helm](https://github.com/goharbor/harbor-helm) chart, these vars can be passed in under ``core.configureUserSettings`` as a JSON string:
+
+```yaml
+core:
+  configureUserSettings: |
+    {
+      "auth_mode": "ldap_auth",
+      "ldap_url": "ldaps://lldap.example.com",
+      "ldap_base_dn": "ou=people,dc=example,dc=com",
+      "ldap_search_dn": "uid=admin,ou=people,dc=example,dc=com",
```
Comment (nitnelave) [inline example_configs/harbor.md:14]: Can you document what permissions this user needs? readonly?

File: example_configs/harbor.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+You can pass environment variables into ``harbor-core`` for auth configuration as documented [here](https://github.com/goharbor/website/blob/release-2.10.0/docs/install-config/configure-system-settings-cli.md#harbor-configuration-items).
```
Comment (nitnelave) [inline example_configs/harbor.md:1]: Can you add a (very) quick word on what is Harbor?

**Review by nitnelave — CHANGES_REQUESTED** [review]

File: example_configs/harbor.md (hunk 1 above)
Comment (kevin7s-io) [reply example_configs/harbor.md]: Done.

File: example_configs/harbor.md (hunk 2 above)
Comment (kevin7s-io) [reply example_configs/harbor.md:3]: Done.

**Review by nitnelave — APPROVED** [review]

File: example_configs/harbor.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      "ldap_group_search_filter": "(objectClass=groupOfUniqueNames)",
+      "ldap_group_attribute_name": "uid"
+    }
+```
+
+> [!IMPORTANT]  
+> ``ldap_search_dn`` needs to be able to bind and search. The ``lldap_strict_readonly`` group is sufficient.
+
+> [!NOTE]  
+> Members of the ``ldap_group_admin_dn`` group will receive harbor admin privledges. Users outside this group or omitting the property will grant standard access.
```
[the remark this answers is not in the document]
Comment (kevin7s-io) [reply example_configs/harbor.md:29]: Good point -- I should clarify this. It's covered in the docs, but the user's groups
 gets imported into harbor and you can assign harbor RBAC based on that group(s).

Screenshot 2024-03-12 at 1 45 01 PM

let me do one more small rev.

[the remark this answers is not in the document]
Comment (kevin7s-io) [reply example_configs/harbor.md]: [this commit](https://github.com/lldap/lldap/pull/862/commits/1ff56eb440ae7eae6441c79035caa385c7b299bf) adds (short) explanation of how harbor handles ldap groups.

**Review by nitnelave — APPROVED** [review]

**Comment by nitnelave** [comment]
Thanks!

**Pull request merged**